config/configmap/inferenceservice.yaml

```diff
@@ -107,6 +107,14 @@ data:
               "lightgbm"
             ],
             "multiModelServer": false
+        },
+        "paddle": {
```

Comment:
@Ruminateer Please add this section to the [e2e test config overlay](https://github.com/kubeflow/kfserving/blob/master/config/overlays/test/configmap/inferenceservice.yaml).